CDP4JsonFileDal/JsonFileDalUtils.cs

```diff
@@ -39,12 +39,12 @@ internal static class JsonFileDalUtils
         /// <summary>
         /// A remark to be included in the exchange header file.
         /// </summary>
-        private const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
+        public const string ExchangeHeaderRemark = "This is an ECSS-E-TM-10-25 exchange file";
```

Comment:
This could be `internal`. no need for `public`.

Reply:
Could be a nice developers discussion ;).
I myself am used to use `public` unless there is a specific reason to use `internal`.
I think that most of our code is build using that way of thinking.
Don't know how others think about this. @samatrhea @alexatrhea ???

Reply:
After some extra thought, These two properties could be left unchanged (neither access level nor name needs a change), as they could only be set in `JsonFileDal.cs` if they are not null (no need to use the default values in the following method, as they are already set):
```
        public void UpdateExchangeFileHeader(Person person, string headerCopyright = JsonFileDalUtils.DefaultHeaderCopyright, string headerRemark = JsonFileDalUtils.ExchangeHeaderRemark)
```